package.json

```diff
@@ -28,10 +28,12 @@
     "handlebars": "4.x.x",
     "hapi": "16.x.x",
     "haute-couture": ">=2.1.0 <3",
+    "highlight.js": "^9.12.0",
```

Comment:
Can you remove this if we're not going to use it?

Reply:
Oops! Totally left that by accident, cleaning up!